src/main/java/org/interledger/core/InterledgerAddress.java

```diff
@@ -0,0 +1,70 @@
+package org.interledger.core;
```

Comment:
This PR proposes that ILP-Java base classes (like InterledgerAddress.java, for example) be located in the `org.interledger.core` package.

The rationale is that something like `org.interledger.ilp` is somewhat redundant.  For example, will the Interledger "community" ever publish something that's not `ilp`?  It seems like the answer is _no_.  

The only counter-point to this line of thinking is if the community (under the auspices of org.interledger) decided to publish an implementation of something like a ledger that doesn't support ILP.  One might argue that in that case, the `org.interledger.ledger` package might need to be distinct from `org.interledger.ilp.ledger`, but I find this type of possibility unlikely, for the primary reason that if somebody wants to be publishing non-ILP components, then probably the `org.interledger` package is not the appropriate place to do that.

Thoughts?